### ISSUE TITLE
Nuxt Dev Server warnings after installing module: Some plugins are not exposing a function and skipped warning

### ISSUE THREAD

**Issue by robertpatrick** [nuxt-modules/supabase#235]
### Version
@nuxtjs/supabase: 1.0.2
nuxt: 3.0.0

### Reproduction Link

https://github.com/robertpatrick/mastering-nuxt-3

### Steps to reproduce

1. `git clone git@github.com:robertpatrick/mastering-nuxt-3.git`
2. `cd mastering-nuxt-3`
3. `npm i`
4. `npm run dev`

### What is Expected?

The dev server to start without warnings...

### What is actually happening?

On running `npm run dev`, I see the following warnings after adding the `@nuxtjs/supabase` module.

```
✔ Nitro built in 496 ms                                         nitro 13:59:11
ℹ Vite client warmed up in 2036ms                                     13:59:11
[warn] [nuxt] Some plugins are not exposing a function and skipped: [ 13:59:15
  {
    name: 'auth-redirect',
    setup: [Function: setup],
    __nuxt_plugin: true
  },
  {                                                                   13:59:07
    name: 'supabase',
    enforce: 'pre',
    setup: [AsyncFunction: setup],
    __nuxt_plugin: true
  }
]
```

**Comment by laurentRichir** [comment]
Hello @robertpatrick 
Did you manage to fix the error? Because I have the same as you but I do not pass my auth
thanks

**Comment by robertpatrick** [comment]
@laurentRichir No.  I ended up having to drop back to an earlier version of supabase to get it to work properly...

**Comment by robmontesinos** [comment]
I got it to work on 0.3.4. Any chance a more current version works?

**Comment by JKausits** [comment]
I found that nuking the node_modules folder, and doing a fresh npm install seemed to have to resolved the issue for me locally.

**Comment by PI-Gorbo** [comment]
I am getting this error, havn't been able to re-create @JKausits experience and looking for a fix.
I am also finding that the `useSupabaseClient()` is returning undefined.
Not sure why, but I can get supabase-js to successfully create a client using the createClient() function.
